obf_creation/rendering_types.xml

```diff
@@ -5240,6 +5240,23 @@
 		<entity_convert pattern="tag_transform" from_tag="denomination" from_value="non-denominational" to_tag1="denomination" to_value1="nondenominational" map="no"/>
 		<entity_convert pattern="tag_transform" from_tag="denomination" from_value="shingon" to_tag1="denomination" to_value1="shingon_shu" map="no"/>
 		<entity_convert pattern="tag_transform" from_tag="religion" if_tag1="denomination" if_value1="jehovahs_witness" poi="no"/>
+
+		<entity_convert pattern="tag_transform" from_tag="religion" from_value="christian" if_tag1="building" if_value1="church" to_tag1="osmand_religious" to_value1="christian_church" to_tag2="osmand_religious_christian_church" to_value2="church" map="no" poi="yes"/>
+		<entity_convert pattern="tag_transform" from_tag="religion" from_value="christian" if_contains_tag1="name" if_contains_value1="church" to_tag1="osmand_religious" to_value1="christian_church" to_tag2="osmand_religious_christian_church" to_value2="church" map="no" poi="yes"/>
+		<entity_convert pattern="tag_transform" from_tag="religion" from_value="christian" if_contains_tag1="name" if_contains_value1="церковь" to_tag1="osmand_religious" to_value1="christian_church" to_tag2="osmand_religious_christian_church" to_value2="church" map="no" poi="yes"/>
+		<entity_convert pattern="tag_transform" from_tag="religion" from_value="christian" if_tag1="building" if_value1="yes" to_tag1="osmand_religious" to_value1="christian_church" to_tag2="osmand_religious_christian_church" to_value2="other" map="no" poi="yes"/>
+
+		<entity_convert pattern="tag_transform" from_tag="religion" from_value="christian" if_tag1="building" if_value1="cathedral" to_tag1="osmand_religious" to_value1="christian_church" to_tag2="osmand_religious_christian_church" to_value2="cathedral" map="no" poi="yes"/>
+		<entity_convert pattern="tag_transform" from_tag="religion" from_value="christian" if_contains_tag1="name" if_contains_value1="cathedral" to_tag1="osmand_religious" to_value1="christian_church" to_tag2="osmand_religious_christian_church" to_value2="cathedral" map="no" poi="yes"/>
```

Comment:
Cathedral should be determined by 
"church:type=cathedral", using `church:type` or `cathedral`

Reply:
This tag is not used for that https://taginfo.openstreetmap.org/keys/church%3Atype#values
cathedral can be determined by building=cathedral or by name